README.md

```diff
@@ -236,6 +252,24 @@ open-attestation document-store revoke --network ropsten --address 0x19f89607b52
 ✔  success   Document/Document Batch with hash 0x0c1a666aa55d17d26412bb57fbed96f40ec5a08e2f995a108faf45429ae3511f has been revoked on 0x19f89607b52268D0A19543e48F790c65750869c6
 ```
 
+## Gsn Capable
+
+```bash
+open-attestation gsn-capable set-paymaster --gsn-capable-address <CONTRACT_ADDRESS> --paymaster-address <PAYMASTER_CONTRACT> [options]
+```
+
+Example - with private key set in `OA_PRIVATE_KEY` environment variable (recommended). [More options](#providing-the-wallet).
+
+```bash
+open-attestation gsn-capable set-paymaster --network ropsten --gsn-capable-address 0x0d3dFdd82FF13Ff06a336e28CABE465B64fD8168 --paymaster-address 0xcB94584760bCA09e9fa7117C4eE966814f17a306
+
+✔  success   Paymaster address 0xcB94584760bCA09e9fa7117C4eE966814f17a306 has been successfully set on 0x0d3dFdd82FF13Ff06a336e28CABE465B64fD8168
+```
+
+### setPaymaster
```

Comment:
what's that ?

Reply:
This is a method to set a paymaster for the receiptant contract who will pay the relayer of the meta transaction! When frontend query it will return this address so we dont ahve to declare any address on the frontend

Reply:
ok, but the doc is missing? there is just a title, coming out of nowhere wth one sentence

Reply:
Ill link back to oa-documentation when its done about how to use it with frontend relayer!